app/models/feature_flag.rb

```diff
@@ -0,0 +1,9 @@
+class FeatureFlag < ApplicationRecord
+  def self.enabled?(name)
+    flag = FeatureFlag.find_by(name: name)
+
+    return false if flag.nil?
+
+    flag.enabled?
```

Comment:
```suggestion
  where(name: name, enabled: true).exists?
```
Minor but could save loading an object, not a blocker though!

Reply:
well tests pass so happy to include this change